assets/js/script.js

```diff
@@ -1,18 +1,20 @@
-// Event listener for click id recipeNavigation and hide .nutritional-analysis-section
+// Event listener for click id recipeNavigation and hide #nutritional-analysis
 $("#recipeNavigation").on("click", function () {
   $(".recipe-search-section").removeClass("d-none");
   $(".randomRecipes").removeClass("d-none");
-  $(".nutritional-analysis-section").addClass("d-none");
+  $("#nutritional-analysis").addClass("d-none");
 });
 
 // Event listener for click id nutritionalNavigation and hide .recipe-search-section
 $("#nutritionalNavigation").on("click", function () {
   $(".recipe-search-section").addClass("d-none");
   $(".randomRecipes").addClass("d-none");
-  $(".nutritional-analysis-section").removeClass("d-none");
+  $("#nutritional-analysis").removeClass("d-none");
 });
 
 // Hide .nutritional-analysis-section on load
 $(function () {
```

Comment:
Delete the entire doc ready function as the class is added to index.html to hide on load:

<section class="section container-fluid d-none" id="nutritional-analysis-container">